features/minimal-printf/README.md

```diff
@@ -0,0 +1,115 @@
+# Minimal printf and snprintf
```

Comment:
Can we add configuration explanation and how can people further optimise the memory use?

Reply:
Done